src/openrct2/actions/MazePlaceTrackAction.hpp

```diff
@@ -93,7 +93,8 @@ DEFINE_GAME_ACTION(MazePlaceTrackAction, GAME_COMMAND_PLACE_MAZE_DESIGN, GameAct
                 { _loc.ToTileStart(), baseHeight, clearanceHeight }, &map_place_non_scenery_clear_func, { 0b1111, 0 },
                 GetFlags(), &clearCost, CREATE_CROSSING_MODE_NONE))
         {
-            return MakeResult(GA_ERROR::NO_CLEARANCE, res->ErrorTitle, gGameCommandErrorText, gCommonFormatArgs);
+            return MakeResult(
+                GA_ERROR::NO_CLEARANCE, std::get<rct_string_id>(res->ErrorTitle), gGameCommandErrorText, gCommonFormatArgs);
```

Comment:
This is not supported in macOS 10.13 and earlier. We need to decide whether to drop support for macOS 10.13 and earlier, or using an alternative.

Reply:
I think we may be able to work around this; it's just a particular signature of the std::variant header that's not supported (a variant, if you will...)